templates/web/thamesmead/about/faq-en-gb.html

```diff
@@ -36,6 +36,7 @@ <h1><a name="faq"></a>[% loc('Frequently Asked Questions') %]</h1>
         <li>Tell us you are worried about someone
         <li>Ask something else
     </ul>
+    <p>Please note that Peabody is not responsible for bin collections. If you have any questions or concerns about bin collections, please contact your local authority. You can contact Bexley <a href="https://www.bexley.gov.uk/services/rubbish-and-recycling/find-your-collection-day-and-report-missed-bin/when-my-rubbish-and-recycling-collected">here</a> or the Royal Borough of Greenwich <a href="https://www.royalgreenwich.gov.uk/info/200171/recycling_and_rubbish">here</a>.</p>
```

Comment:
The use of "here" as a link isn't good for accessibility - reword it to perhaps
`please contact your local authority, <a href="https://www.bexley.gov.uk/services/rubbish-and-recycling/find-your-collection-day-and-report-missed-bin/when-my-rubbish-and-recycling-collected">Bexley</a> or the <a href="https://www.royalgreenwich.gov.uk/info/200171/recycling_and_rubbish">Royal Borough of Greenwich</a>`?